xml/common_intro_lifecycle_support_i.xml

```diff
@@ -0,0 +1,214 @@
+      been identified by Level&nbsp;2 Support.
+     </para>
+    </listitem>
+   </varlistentry>
+  </variablelist>
+  <para>
+   For contracted customers and partners, &productname; is delivered with L3
+   support for all packages, except for the following:
+  </para>
+  <itemizedlist>
+   <listitem>
+    <para>
+     Technology Previews
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Sound, graphics, fonts and artwork.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Packages that require an additional customer contract.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Some packages shipped as part of the module <emphasis>Workstation
+     Extension</emphasis> are L2-supported only.
+    </para>
+   </listitem>
+   <listitem>
+    <para>
+     Packages with names ending in <package>-devel</package> (containing header
+     files and similar developer resources) will only be supported together
+     with their main packages.
+    </para>
+   </listitem>
+  </itemizedlist>
+  <para>
+   &suse; will only support the usage of original packages. That is, packages
+   that are unchanged and not recompiled.
+  </para>
+ </sect2>
+
+ <sect2>
+  <title>Technology Previews</title>
+   <para>
+    Technology previews are packages, stacks, or features delivered by &suse;
+    to provide insights into upcoming innovations. These previews are still in
+    development, but are included for your convenience to give you the chance to
+    test new technologies within an enterprise environment. We would appreciate
+    your feedback! If you test a technology preview, please contact your
+    &suse; representative and let them know about your experience and use cases.
+    Your input is helpful for future development.
+   </para>
+   <para>
+    However, technology previews come with the following limitations:
+   </para>
+   <itemizedlist>
+    <listitem>
+     <para>
+      Technology previews may be functionally incomplete, unstable, or in other ways
+      <emphasis>not</emphasis> suitable for production use.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Technology previews are <emphasis>not</emphasis> supported.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Technology previews may only be available for specific hardware
+      architectures.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Details and functionality of technology previews are subject to change.
+      As a result, upgrading to subsequent releases of a technology preview may
+      be impossible and require a fresh installation.
+     </para>
+    </listitem>
+    <listitem>
+     <para>
+      Technology previews can be dropped at any time (for example, if &suse;
+      discovers that a preview does not meet the customer or market needs, or does
+      not prove to comply with enterprise standards). &suse; does not commit to
```

Comment:
this is a very long paren. maybe just make this an independent sentence?

Reply:
Done. Thanks for the feedback, Stefan!